modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,73 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying virtual router conflicts
+
+The Openshift cluster Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router-id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have multiple clusters using the same network, you should check to see which VIPs the installation will choose to avoid conflicts.
+
+To discover the VIPs that will be chosen, you can use the runtimecfg utility.
+
+.Procedure
+
+. Identify the release being used by the installer:
++
+[source, terminal]
+----
+$ ./openshift-install version|grep 'release image'
+----
++
+.Example output
+[source, terminal]
+----
+release image registry.ci.openshift.org/ocp/release@sha256:9aa0dce0b119d1d3f284df2cd30394a23e2d97f4b3e3853503a655caf90219a5
+----
++
+. Display information about the release:
++
+[source, terminal]
+----
+$ oc adm release info --registry-config pull-secret.txt registry.ci.openshift.org/ocp/release@sha256:9aa0dce0b119d1d3f284df2cd30394a23e2d97f4b3e3853503a655caf90219a5 -o json | jq -r '.references.spec.tags[] | select(.name == "baremetal-runtimecfg") | .from.name'
+----
++
+.Example output
+[source, terminal]
+----
+quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b717191f780ebc5a9cce440993f51e0a2b6ef3eab3c450185996b4c79189506b
+----
++
++
+. Run the runtimecfg utility with the cluster name, in this example `cnf10`:
++
+[source, terminal]
+----
+$ podman run --authfile /home/dev-scripts/pull_secret.json quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b717191f780ebc5a9cce440993f51e0a2b6ef3eab3c450185996b4c79189506b vr-ids cnf10
+----
++
+.Example output
+[source, terminal]
+----
+APIVirtualRouterID: 147
+DNSVirtualRouterID: 158
+IngressVirtualRouterID: 2
+----
++
+. Run the utility again using a different cluster name, in this example `cnf11`:
++
+[source, terminal]
+----
+$ podman run --authfile /home/dev-scripts/pull_secret.json quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b717191f780ebc5a9cce440993f51e0a2b6ef3eab3c450185996b4c79189506b vr-ids cnf11
+----
++
+.Example output
+[source, terminal]
+----
+APIVirtualRouterID: 228
+DNSVirtualRouterID: 239
+IngressVirtualRouterID: 147
+----
++
+In the example output above you can see that installing two clusters in the same multicast domain with names cnf10 and cnf11 will lead to a conflict with the `APIVirtualRouterID` and the `IngressVirtrualRouterID`.
```

Comment:
I think you should add here:
"""
In the example output above you can see that installing two clusters in the same multicast domain with the cluster names cnf10 and cnf11 will lead to a conflict with the `APIVirtualRouterID` and the `IngressVirtrualRouterID`.
Therefore one of the cluster names needs to be changed and the process needs to be repeated till there is no conflict
"""
@janosdebugs What do you think?

Reply:
*until

Reply:
Done.